create.go

```diff
@@ -115,6 +119,14 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 }
 
 func getConfigs(bundlePath, containerID, console string) (vc.PodConfig, specs.Spec, error) {
+	if bundlePath == "" {
```

Comment:
Not sure about checking those values again in an internal functions when we have things like validCreateParams() that seems like the place to have those checks.